.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -129,7 +129,6 @@ jobs:
         id: cilium-stable-config
         uses: ./.github/actions/cilium-config
         with:
-          image-tag: ${{ steps.vars.outputs.downgrade_version }}
```

Comment:
Won't this cause downgrade to `cilium/cilium-ci:latest`, which is AFAIU dev version of 1.15?

I am not sure if we want to test downgrade between PR version and current main instead of PR version and previous stable version.

Reply:
Without `image-tag`, cilium-config action gives installation flags without specific image repos and tags. Therefore, when `CILIUM_CLI_MODE=helm ./cilium-cli install` runs, it's `--chart-directory` which has a say in which image is using.

In this case, `chart-dir` is set to `./cilium-${{ steps.vars.outputs.downgrade_version }}/install/kubernetes/cilium/`, which will be checked out from v1.14 branch by this task: https://github.com/cilium/cilium/blob/cb6349bbce577484c3635816a9747e44369a1c3e/.github/workflows/tests-ipsec-upgrade.yaml#L173-L178

We can confirm by looking at the successful CI task:  https://github.com/cilium/cilium/actions/runs/6119666881/job/16610076122, the task `Downgrade Cilium to v1.14 & Test (3)` has logs demonstrating it was downgraded to v1.14.1

So I assume it causes downgrade to v1.14 as expected?

Reply:
I missed that, thanks for explaining!